lib/SIL/Utils/DynamicCasts.cpp

```diff
@@ -881,6 +888,20 @@ swift::classifyDynamicCast(ModuleDecl *M,
   return DynamicCastFeasibility::WillFail;
 }
 
+bool swift::matchesActorIsolation(ProtocolConformance *conformance, SILFunction *inFunction) {
```

Comment:
Per the previous comment, I recommend passing a `ProtocolConformanceRef` in here so the callers don't do the isConcrete/getConcrete dance.